### ISSUE TITLE
Error when compressing large file

### ISSUE THREAD

**Issue by milankl** [milankl/ZfpCompression.jl#9]
```julia
ERROR: LoadError: InexactError: trunc(UInt32, 7875749720)
Stacktrace:
 [1] throw_inexacterror(::Symbol, ::Type{UInt32}, ::Int64) at ./boot.jl:558
 [2] checked_trunc_uint at ./boot.jl:588 [inlined]
 [3] toUInt32 at ./boot.jl:672 [inlined]
 [4] UInt32 at ./boot.jl:712 [inlined]
 [5] convert at ./number.jl:7 [inlined]
 [6] cconvert at ./essentials.jl:388 [inlined]
 [7] stream_open at /home/kloewer/.julia/packages/ZfpCompression/rpgbM/src/zfp.jl:195 [inlined]
 [8] zfp_compress(::Array{Float32,4}; write_header::Bool, kws::Base.Iterators.Pairs{Symbol,Int64,Tuple{Symbol},NamedTuple{(:precision,),Tuple{Int64}}}) at /home/kloewer/.julia/packages/ZfpCompression/rpgbM/src/zfp.jl:257
```

**Comment by milankl** [comment]
size was 14742MB

**Comment by milankl** [comment]
resolved with #10
